doc/source/conf.py

```diff
@@ -36,7 +36,7 @@ def __getattr__(cls, name):
     "gym.spaces",
     "kubernetes",
     "psutil",
-    "ray._raylet",
+    # "ray._raylet",
```

Comment:
Without this ObjectRef and PlacementGroupID are mocked out, but I am not sure if it is a good idea. cc @richardliaw ?

Reply:
Does it compile + can you scan the ray core api reference to see if it breaks anything?

Reply:
It doesn't seem to break anything. I removed it from here.